django_project/blog/urls.py

```diff
@@ -6,7 +6,7 @@
 from .views import (
     HomeView,
     CreatePostView,
-    create_comment,
+    CreateCommentView,
```

Comment:
I resolved this in the other PR. 
This view is redundant, since the PollDetailView handles it. 
If it's to be used, then the part labelled "comment" in the PollDetailView should be deleted

Reply:
I think it makes more sense to have it in a separate view since PostDetailView is already doing many things.

I removed the `comment` code from the PostDetailView